service/history/tasks/delete_visibility_task.go

```diff
@@ -81,3 +82,7 @@ func (t *DeleteExecutionVisibilityTask) GetCategory() Category {
 func (t *DeleteExecutionVisibilityTask) GetType() enumsspb.TaskType {
 	return enumsspb.TASK_TYPE_VISIBILITY_DELETE_EXECUTION
 }
+
+func (t *DeleteExecutionVisibilityTask) GetCloseVisibilityTaskID() int64 {
```

Comment:
Sorry for another late comment.

Methods in files of this package is for implementing the task.Task interface{}. I would prefer to be consistent with the rest of the files. I don't have very strong opinion though.

I am not sure how useful this method is actually. The rest of the codebase uses `Task` interface (except when generating the delete task) and to get the close taskID, type assertion is needed. But if type assertion is used then we can directly assess the exported field without a method like what you did in `visibilityDeleteTaskToProto` in `task_serializer.go`